protobufs/livekit_models.proto

```diff
@@ -223,8 +223,10 @@ message DataPacket {
     ActiveSpeakerUpdate speaker = 3 [deprecated=true];
     SipDTMF sip_dtmf = 6;
     Transcription transcription = 7;
+    TimeSyncRequest time_sync_request = 8;
```

Comment:
I'm wondering if we could omit the request and always send NTP time when the data channel is first established?

Reply:
The client needs to estimate the round trip time, which is easier with a requrest/response flow.